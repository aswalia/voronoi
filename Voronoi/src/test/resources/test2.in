lft:
lft:
lft:
lft:
lft:
lft:
lft:
point:(0.0,5.0)
lft:
point:(1.0,7.0)
lft:
point:(1.0,19.0)
point:(2.0,16.0)
point:(3.0,40.0)
lft:
point:(4.0,24.0)
point:(4.0,34.0)
point:(4.0,37.0)
lft:
lft:
lft:
point:(4.0,40.0)
point:(5.0,40.0)
point:(5.0,48.0)
point:(6.0,33.0)
lft:
point:(6.0,45.0)
lft:
lft:
point:(8.0,0.0)
point:(8.0,41.0)
lft:
point:(10.0,17.0)
lft:
point:(10.0,33.0)
lft:
point:(11.0,24.0)
lft:
lft:
point:(11.0,35.0)
point:(14.0,30.0)
point:(14.0,48.0)
lft:
point:(15.0,33.0)
lft:
lft:
lft:
lft:
lft:
lft:
point:(16.0,17.0)
point:(18.0,19.0)
point:(20.0,11.0)
lft:
point:(21.0,19.0)
lft:
point:(21.0,35.0)
point:(22.0,28.0)
lft:
point:(25.0,30.0)
lft:
lft:
point:(26.0,37.0)
lft:
point:(27.0,20.0)
lft:
point:(28.0,11.0)
point:(29.0,44.0)
lft:
point:(32.0,26.0)
lft:
lft:
point:(34.0,49.0)
point:(37.0,23.0)
point:(38.0,40.0)
point:(39.0,6.0)
lft:
lft:
lft:
lft:
lft:
point:(39.0,20.0)
lft:
point:(39.0,47.0)
lft:
point:(40.0,20.0)
point:(41.0,10.0)
point:(42.0,32.0)
lft:
lft:
point:(42.0,42.0)
point:(43.0,31.0)
point:(44.0,10.0)
point:(44.0,21.0)
lft:
lft:
point:(47.0,5.0)
point:(47.0,7.0)
lft:
point:(47.0,48.0)
point:(48.0,19.0)
