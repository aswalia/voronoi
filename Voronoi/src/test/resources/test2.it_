node
rgt-point:(27.0,47.0)
node
node
rgt-point:(6.0,11.0)
node
rgt-point:(3.0,38.0)
rgt-point:(2.0,28.0)
node-point:(0.0,3.0) (2.0,14.0)
rgt-point:(6.0,10.0)
node-point:(4.0,0.0) (5.0,11.0)
node
node-point:(7.0,24.0) (7.0,38.0)
lft-point:(8.0,5.0)
node-point:(8.0,32.0) (9.0,34.0)
lft-point:(12.0,12.0)
node
node
node-point:(13.0,31.0) (15.0,6.0)
node-point:(15.0,13.0) (15.0,40.0)
lft-point:(17.0,7.0)
node
node-point:(18.0,6.0) (21.0,12.0)
lft-point:(21.0,29.0)
node
node-point:(22.0,9.0) (22.0,17.0)
lft-point:(22.0,45.0)
node-point:(25.0,27.0) (26.0,32.0)
node
rgt-point:(42.0,29.0)
node
lft-point:(29.0,20.0)
lft-point:(29.0,44.0)
node-point:(30.0,6.0) (30.0,43.0)
lft-point:(31.0,30.0)
node
lft-point:(34.0,36.0)
lft-point:(34.0,40.0)
node-point:(35.0,13.0) (35.0,32.0)
lft-point:(37.0,11.0)
lft-point:(37.0,39.0)
node-point:(37.0,47.0) (42.0,3.0)
node
node-point:(43.0,32.0) (44.0,11.0)
lft-point:(44.0,28.0)
node
rgt-point:(47.0,8.0)
rgt-point:(46.0,42.0)
node-point:(44.0,38.0) (46.0,6.0)
node-point:(48.0,49.0) (49.0,24.0)
