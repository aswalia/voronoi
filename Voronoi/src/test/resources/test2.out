Vor:
(36.5,58.5) (32.0,105.75)
lft and rgt: (27.0,47.0) (48.0,49.0)
(32.0,105.75) (32.0,48.166666666666664)
lft and rgt: (27.0,47.0) (37.0,47.0)
(32.0,48.166666666666664) (25.394736842105264,43.76315789473684)
lft and rgt: (27.0,47.0) (29.0,44.0)
(-148.5,478.5) (25.394736842105264,43.76315789473684)
lft and rgt: (22.0,45.0) (27.0,47.0)
(25.394736842105264,43.76315789473684) (24.67241379310345,38.706896551724135)
lft and rgt: (22.0,45.0) (29.0,44.0)
(24.67241379310345,38.706896551724135) (21.833333333333332,37.833333333333336)
lft and rgt: (22.0,45.0) (26.0,32.0)
(5.615384615384615,60.53846153846154) (21.833333333333332,37.833333333333336)
lft and rgt: (15.0,40.0) (22.0,45.0)
(21.833333333333332,37.833333333333336) (20.335616438356166,35.773972602739725)
lft and rgt: (15.0,40.0) (26.0,32.0)
(20.335616438356166,35.773972602739725) (18.144736842105264,34.578947368421055)
lft and rgt: (15.0,40.0) (21.0,29.0)
(13.357142857142858,35.642857142857146) (18.144736842105264,34.578947368421055)
lft and rgt: (13.0,31.0) (15.0,40.0)
(18.144736842105264,34.578947368421055) (15.25,23.0)
lft and rgt: (13.0,31.0) (21.0,29.0)
(9.74390243902439,27.71951219512195) (15.25,23.0)
lft and rgt: (7.0,24.0) (13.0,31.0)
(15.25,23.0) (15.43063583815029,22.49421965317919)
lft and rgt: (7.0,24.0) (21.0,29.0)
(15.43063583815029,22.49421965317919) (14.885321100917432,21.325688073394495)
lft and rgt: (7.0,24.0) (22.0,17.0)
(14.88532110091743,21.325688073394495) (11.402439024390244,18.79268292682927)
lft and rgt: (7.0,24.0) (15.0,13.0)
(11.402439024390244,18.79268292682927) (8.019480519480519,17.383116883116884)
lft and rgt: (7.0,24.0) (12.0,12.0)
(7.681818181818182,17.40909090909091) (8.019480519480519,17.383116883116884)
lft and rgt: (6.0,11.0) (7.0,24.0)
(8.019480519480519,17.383116883116884) (9.166666666666666,10.5)
lft and rgt: (6.0,11.0) (12.0,12.0)
(5.5,10.5) (9.166666666666666,10.5)
lft and rgt: (6.0,10.0) (6.0,11.0)
(9.166666666666666,10.5) (9.794117647058824,8.617647058823529)
lft and rgt: (6.0,10.0) (12.0,12.0)
(9.794117647058822,8.617647058823529) (3.477272727272727,6.090909090909091)
lft and rgt: (6.0,10.0) (8.0,5.0)
(2.1923076923076925,7.1923076923076925) (3.477272727272727,6.090909090909091)
lft and rgt: (0.0,3.0) (6.0,10.0)
(3.4772727272727275,6.090909090909091) (3.96875,4.125)
lft and rgt: (0.0,3.0) (8.0,5.0)
(3.96875,4.125) (0.5,-0.5)
lft and rgt: (0.0,3.0) (4.0,0.0)
(3.96875,4.125) (12.693548387096774,-2.854838709677419)
lft and rgt: (4.0,0.0) (8.0,5.0)
(12.693548387096774,-2.854838709677419) (16.5,-9.833333333333334)
lft and rgt: (4.0,0.0) (15.0,6.0)
(16.5,-9.833333333333332) (24.0,-27.333333333333332)
lft and rgt: (4.0,0.0) (18.0,6.0)
(24.0,-27.333333333333336) (25.939999999999998,-35.74)
lft and rgt: (4.0,0.0) (30.0,6.0)
(25.94,-35.74) (24.5,-17.5)
lft and rgt: (4.0,0.0) (42.0,3.0)
(36.092592592592595,4.87037037037037) (25.939999999999998,-35.74)
lft and rgt: (30.0,6.0) (42.0,3.0)
(40.5,7.625) (36.092592592592595,4.87037037037037)
lft and rgt: (37.0,11.0) (42.0,3.0)
(33.083333333333336,9.083333333333334) (36.092592592592595,4.87037037037037)
lft and rgt: (30.0,6.0) (37.0,11.0)
(40.5,16.5) (33.083333333333336,9.083333333333334)
lft and rgt: (35.0,13.0) (37.0,11.0)
(27.983870967741936,12.725806451612904) (33.083333333333336,9.083333333333334)
lft and rgt: (30.0,6.0) (35.0,13.0)
(27.96153846153846,13.038461538461538) (27.983870967741936,12.725806451612904)
lft and rgt: (21.0,12.0) (35.0,13.0)
(27.983870967741936,12.725806451612904) (27.92857142857143,12.642857142857142)
lft and rgt: (21.0,12.0) (30.0,6.0)
(27.92857142857143,12.642857142857142) (18.852941176470587,9.617647058823529)
lft and rgt: (21.0,12.0) (22.0,9.0)
(17.676470588235293,10.558823529411764) (18.852941176470587,9.617647058823529)
lft and rgt: (17.0,7.0) (21.0,12.0)
(18.852941176470587,9.617647058823529) (19.357142857142858,8.357142857142858)
lft and rgt: (17.0,7.0) (22.0,9.0)
(19.357142857142858,8.357142857142858) (16.5,5.5)
lft and rgt: (17.0,7.0) (18.0,6.0)
(14.5,9.5) (16.5,5.5)
lft and rgt: (15.0,6.0) (17.0,7.0)
(16.5,5.5) (16.5,-9.833333333333332)
lft and rgt: (15.0,6.0) (18.0,6.0)
(17.676470588235293,10.558823529411764) (14.5,9.5)
lft and rgt: (15.0,13.0) (17.0,7.0)
(14.5,9.5) (14.5,9.5)
lft and rgt: (15.0,6.0) (15.0,13.0)
(11.402439024390244,18.79268292682927) (14.5,9.5)
lft and rgt: (12.0,12.0) (15.0,13.0)
(14.5,9.5) (11.166666666666666,7.833333333333333)
lft and rgt: (12.0,12.0) (15.0,6.0)
(9.794117647058824,8.617647058823529) (11.166666666666666,7.833333333333333)
lft and rgt: (8.0,5.0) (12.0,12.0)
(11.166666666666666,7.833333333333333) (12.693548387096774,-2.85483870967742)
lft and rgt: (8.0,5.0) (15.0,6.0)
(19.357142857142858,8.357142857142858) (24.0,2.166666666666667)
lft and rgt: (18.0,6.0) (22.0,9.0)
(24.0,2.166666666666667) (24.0,-27.33333333333333)
lft and rgt: (18.0,6.0) (30.0,6.0)
(27.92857142857143,12.642857142857142) (24.0,2.166666666666667)
lft and rgt: (22.0,9.0) (30.0,6.0)
(18.43548387096774,15.112903225806452) (17.676470588235293,10.558823529411764)
lft and rgt: (15.0,13.0) (21.0,12.0)
(18.435483870967744,15.112903225806452) (27.75,13.25)
lft and rgt: (21.0,12.0) (22.0,17.0)
(27.75,13.25) (27.96153846153846,13.038461538461538)
lft and rgt: (21.0,12.0) (29.0,20.0)
(24.07377049180328,21.827868852459016) (27.75,13.25)
lft and rgt: (22.0,17.0) (29.0,20.0)
(29.72222222222222,25.055555555555557) (24.07377049180328,21.827868852459016)
lft and rgt: (25.0,27.0) (29.0,20.0)
(20.456521739130434,22.91304347826087) (24.07377049180328,21.827868852459016)
lft and rgt: (22.0,17.0) (25.0,27.0)
(23.90909090909091,29.818181818181817) (20.456521739130434,22.913043478260867)
lft and rgt: (21.0,29.0) (25.0,27.0)
(20.456521739130434,22.91304347826087) (15.430635838150287,22.49421965317919)
lft and rgt: (21.0,29.0) (22.0,17.0)
(23.90909090909091,29.818181818181817) (27.72222222222222,29.055555555555557)
lft and rgt: (25.0,27.0) (26.0,32.0)
(27.72222222222222,29.055555555555557) (29.72222222222222,25.055555555555557)
lft and rgt: (25.0,27.0) (31.0,30.0)
(29.833333333333332,34.333333333333336) (27.72222222222222,29.055555555555557)
lft and rgt: (26.0,32.0) (31.0,30.0)
(29.833333333333332,34.333333333333336) (31.833333333333332,33.333333333333336)
lft and rgt: (31.0,30.0) (34.0,36.0)
(31.833333333333332,33.333333333333336) (36.07692307692308,24.846153846153847)
lft and rgt: (31.0,30.0) (35.0,32.0)
(36.07692307692308,24.846153846153847) (35.982142857142854,23.803571428571427)
lft and rgt: (31.0,30.0) (42.0,29.0)
(29.72222222222222,25.055555555555557) (35.982142857142854,23.803571428571427)
lft and rgt: (29.0,20.0) (31.0,30.0)
(35.982142857142854,23.803571428571427) (37.672413793103445,21.362068965517242)
lft and rgt: (29.0,20.0) (42.0,29.0)
(37.672413793103445,21.362068965517242) (27.96153846153846,13.038461538461538)
lft and rgt: (29.0,20.0) (35.0,13.0)
(37.672413793103445,21.362068965517242) (39.11538461538461,20.73076923076923)
lft and rgt: (35.0,13.0) (42.0,29.0)
(39.11538461538461,20.73076923076923) (41.166666666666664,19.5)
lft and rgt: (35.0,13.0) (44.0,28.0)
(41.166666666666664,19.5) (40.5,16.5)
lft and rgt: (35.0,13.0) (44.0,11.0)
(41.166666666666664,19.5) (41.3,19.5)
lft and rgt: (44.0,11.0) (44.0,28.0)
(41.3,19.5) (51.55555555555556,15.555555555555555)
lft and rgt: (44.0,11.0) (49.0,24.0)
(51.55555555555556,15.555555555555555) (44.166666666666664,8.166666666666666)
lft and rgt: (44.0,11.0) (47.0,8.0)
(44.166666666666664,8.166666666666666) (41.92307692307692,7.269230769230769)
lft and rgt: (44.0,11.0) (46.0,6.0)
(40.5,7.625) (41.92307692307692,7.269230769230769)
lft and rgt: (42.0,3.0) (44.0,11.0)
(41.92307692307692,7.269230769230769) (45.5,2.5)
lft and rgt: (42.0,3.0) (46.0,6.0)
(40.5,16.5) (40.5,7.625)
lft and rgt: (37.0,11.0) (44.0,11.0)
(44.166666666666664,8.166666666666666) (47.5,6.5)
lft and rgt: (46.0,6.0) (47.0,8.0)
(51.55555555555556,15.555555555555555) (56.0,15.0)
lft and rgt: (47.0,8.0) (49.0,24.0)
(51.25,31.9375) (41.3,19.5)
lft and rgt: (44.0,28.0) (49.0,24.0)
(53.18181818181818,33.38636363636364) (51.25,31.9375)
lft and rgt: (43.0,32.0) (49.0,24.0)
(51.25,31.9375) (43.785714285714285,30.071428571428573)
lft and rgt: (43.0,32.0) (44.0,28.0)
(39.0,31.666666666666668) (43.785714285714285,30.071428571428573)
lft and rgt: (42.0,29.0) (43.0,32.0)
(43.785714285714285,30.071428571428573) (39.11538461538461,20.73076923076923)
lft and rgt: (42.0,29.0) (44.0,28.0)
(39.0,34.642857142857146) (39.0,31.666666666666668)
lft and rgt: (35.0,32.0) (43.0,32.0)
(39.0,31.666666666666668) (36.07692307692308,24.846153846153847)
lft and rgt: (35.0,32.0) (42.0,29.0)
(40.08139534883721,35.56976744186046) (39.0,34.642857142857146)
lft and rgt: (37.0,39.0) (43.0,32.0)
(38.1,34.9) (39.0,34.642857142857146)
lft and rgt: (35.0,32.0) (37.0,39.0)
(35.0,38.0) (38.1,34.9)
lft and rgt: (34.0,36.0) (37.0,39.0)
(38.1,34.9) (31.833333333333332,33.333333333333336)
lft and rgt: (34.0,36.0) (35.0,32.0)
(36.666666666666664,43.0) (35.0,38.0)
lft and rgt: (34.0,40.0) (37.0,39.0)
(29.375,38.0) (35.0,38.0)
lft and rgt: (34.0,36.0) (34.0,40.0)
(33.986486486486484,44.148648648648646) (29.375,38.0)
lft and rgt: (30.0,43.0) (34.0,40.0)
(29.375,38.0) (28.305555555555557,37.388888888888886)
lft and rgt: (30.0,43.0) (34.0,36.0)
(24.7,38.7) (28.305555555555557,37.388888888888886)
lft and rgt: (26.0,32.0) (30.0,43.0)
(28.305555555555557,37.388888888888886) (29.833333333333332,34.333333333333336)
lft and rgt: (26.0,32.0) (34.0,36.0)
(32.59090909090909,46.59090909090909) (24.7,38.7)
lft and rgt: (29.0,44.0) (30.0,43.0)
(24.67241379310345,38.706896551724135) (24.7,38.7)
lft and rgt: (26.0,32.0) (29.0,44.0)
(32.59090909090909,46.59090909090909) (33.986486486486484,44.148648648648646)
lft and rgt: (30.0,43.0) (37.0,47.0)
(32.0,48.166666666666664) (32.59090909090909,46.59090909090909)
lft and rgt: (29.0,44.0) (37.0,47.0)
(33.986486486486484,44.148648648648646) (36.666666666666664,43.0)
lft and rgt: (34.0,40.0) (37.0,47.0)
(36.666666666666664,43.0) (40.666666666666664,43.0)
lft and rgt: (37.0,39.0) (37.0,47.0)
(40.666666666666664,43.0) (41.0,42.0)
lft and rgt: (37.0,39.0) (46.0,42.0)
(41.0,42.0) (40.08139534883721,35.56976744186046)
lft and rgt: (37.0,39.0) (44.0,38.0)
(41.0,42.0) (56.125,34.4375)
lft and rgt: (44.0,38.0) (46.0,42.0)
(56.125,34.4375) (53.18181818181818,33.38636363636364)
lft and rgt: (44.0,38.0) (49.0,24.0)
(40.08139534883721,35.56976744186046) (53.18181818181818,33.38636363636363)
lft and rgt: (43.0,32.0) (44.0,38.0)
(74.81578947368422,37.55263157894737) (56.125,34.4375)
lft and rgt: (46.0,42.0) (49.0,24.0)
(61.0,37.0) (74.8157894736842,37.55263157894737)
lft and rgt: (48.0,49.0) (49.0,24.0)
(42.73287671232877,46.71917808219178) (74.81578947368422,37.55263157894737)
lft and rgt: (46.0,42.0) (48.0,49.0)
(32.0,105.75) (42.73287671232877,46.71917808219178)
lft and rgt: (37.0,47.0) (48.0,49.0)
(42.73287671232877,46.71917808219178) (40.666666666666664,43.0)
lft and rgt: (37.0,47.0) (46.0,42.0)
(20.335616438356166,35.773972602739725) (23.90909090909091,29.818181818181817)
lft and rgt: (21.0,29.0) (26.0,32.0)
(14.88532110091743,21.325688073394495) (18.43548387096774,15.112903225806452)
lft and rgt: (15.0,13.0) (22.0,17.0)
(5.5,10.5) (2.1923076923076925,7.1923076923076925)
lft and rgt: (5.0,11.0) (6.0,10.0)
(-0.26923076923076916,8.73076923076923) (2.1923076923076925,7.1923076923076925)
lft and rgt: (0.0,3.0) (5.0,11.0)
(5.5,14.5) (-0.26923076923076916,8.73076923076923)
lft and rgt: (2.0,14.0) (5.0,11.0)
(-67.75,21.0) (-0.2692307692307694,8.73076923076923)
lft and rgt: (0.0,3.0) (2.0,14.0)
(-67.75,21.0) (0.5,21.0)
lft and rgt: (2.0,14.0) (2.0,28.0)
(0.5,21.0) (7.681818181818182,17.40909090909091)
lft and rgt: (2.0,14.0) (7.0,24.0)
(7.681818181818182,17.40909090909091) (5.5,14.5)
lft and rgt: (2.0,14.0) (6.0,11.0)
(6.2272727272727275,28.15909090909091) (0.5,21.0)
lft and rgt: (2.0,28.0) (7.0,24.0)
(6.2272727272727275,28.15909090909091) (9.74390243902439,27.71951219512195)
lft and rgt: (7.0,24.0) (8.0,32.0)
(3.0357142857142856,32.94642857142857) (6.2272727272727275,28.15909090909091)
lft and rgt: (2.0,28.0) (8.0,32.0)
(5.0,34.583333333333336) (3.0357142857142856,32.94642857142857)
lft and rgt: (3.0,38.0) (8.0,32.0)
(-2.5,33.5) (3.0357142857142856,32.94642857142857)
lft and rgt: (2.0,28.0) (3.0,38.0)
(5.25,34.625) (5.0,34.583333333333336)
lft and rgt: (7.0,38.0) (8.0,32.0)
(5.0,61.85714285714286) (5.0,34.583333333333336)
lft and rgt: (3.0,38.0) (7.0,38.0)
(5.0,61.85714285714286) (5.615384615384617,60.53846153846153)
lft and rgt: (7.0,38.0) (22.0,45.0)
(5.615384615384615,60.53846153846154) (11.333333333333334,37.666666666666664)
lft and rgt: (7.0,38.0) (15.0,40.0)
(11.333333333333334,37.666666666666664) (5.25,34.625)
lft and rgt: (7.0,38.0) (9.0,34.0)
(11.333333333333334,37.666666666666664) (13.357142857142858,35.642857142857146)
lft and rgt: (9.0,34.0) (15.0,40.0)
(13.357142857142858,35.642857142857146) (10.590909090909092,31.954545454545453)
lft and rgt: (9.0,34.0) (13.0,31.0)
(5.25,34.625) (10.59090909090909,31.954545454545453)
lft and rgt: (8.0,32.0) (9.0,34.0)
(10.590909090909092,31.954545454545453) (9.74390243902439,27.71951219512195)
lft and rgt: (8.0,32.0) (13.0,31.0)
(-148.5,478.5) (5.0,61.85714285714286)
lft and rgt: (3.0,38.0) (22.0,45.0)
(-11.5,16.5) (-67.75,21.0)
lft and rgt: (0.0,3.0) (2.0,28.0)
(5.5,14.5) (5.5,10.5)
lft and rgt: (5.0,11.0) (6.0,11.0)
(10.5,54.5) (-148.5,478.5)
lft and rgt: (3.0,38.0) (27.0,47.0)

